.github/workflows/error-ref-publisher.yaml

```diff
@@ -0,0 +1,55 @@
+name: Error Code Reference Publisher
```

Comment:
@Shreyas220 I've updated the workflow in meshkit repo a bit. Will you be able to follow that and update this one too? https://github.com/DelusionalOptimist/meshkit/blob/error-codes-final/.github/workflows/error-codes-updater.yaml
Thanks.